starter-code/index.html

```diff
@@ -42,11 +42,18 @@ <h1>Pizza Builder</h1>
     <!-- Price -->
     <aside class="panel price">
       <h2>Your pizza's price</h2>
-      <b><!-- Put the base price of the pizza here --> cheese pizza</b>
+      <b>
+        <!-- Put the base price of the pizza here -->cheese pizza</b>
       <ul>
+        <li v-for="ingredient in filteredIngredients">
+          <span v-if="isSelected(ingredient.name)">{{ingredient.name}} ${{ingredient.price}}</span>
```

Comment:
If you already filtered them with the computed prop, you don't need the extra check of 
`v-if="isSelected(ingredient.name)"`